learn/janessa/IntakeCode/src/main/java/frc/robot/Robot.java

```diff
@@ -43,11 +43,11 @@
 
   private SpeedControllerGroup magazine;
 
-  private CANSparkMax leftShooter;
-  private CANSparkMax rightShooter;
+  // private CANSparkMax leftShooter;
+  // private CANSparkMax rightShooter;
```

Comment:
Hey, why are you modifying code frrom @janessacodes  shouldn't you be working on your own code?